### DISCUSSION TITLE
Cabinets

### DISCUSSION THREAD

**Discussion by WilsonBradley** [curif/AgeOfJoy-2022.1#20] [Ideas]
Not sure what type of cabinets are compatible, 

Below are just ideas.
However, there might be a GOLD Mine of free available Pre-made assets out on the internet.
For example, look up [New Retro Arcade] on steam (Win PC), might be able to get alot of cabinets and graphics.
https://steamcommunity.com/app/465780/screenshots/

**Comment by WilsonBradley** [comment]
The game sounds, graphics, posters, etc.. are available. 

It seems NRA they used looped arcade sounds for each arcade to give volume control when going closer/farther.
Have looped video / sound for games would probably would help in processing resources. Play a recorded sound until actual emulator is launched.

**Comment by WilsonBradley** [comment]
Awesome, got the test cabinet to work in the workroom. 
It's joust.
Notes:
1.) joust screen needs to be adjusted to landscape from current Portrait layout. ? think I found it in description.yaml
 2.) Note for joust: First time needs to have quarter put in and initialized. After leaving and coming back, it works.
3.) Seriously need ability to assign buttons.

Will work on designing cabinet / assets/ video/sound once it's playable.

**Reply by curif** [reply]
Great, It's good to see that you can create your own cabinets and test them in the testing area. It requires some skills, but once you've done the firsts ones the others come easily.

I just upload a preview of the cabinet packs 4 and 5. You can find them in the Release page here in GitHub: https://github.com/curif/AgeOfJoy-2022.1/releases/tag/v0.2.0-rc0

As you can see in the description.yaml, the cabinet model is included in the cabinet asset with the rest of the files.

Related with your previous message, the sound has some improvements too. It's louder as you get closer to the cabinet.

**Comment by WilsonBradley** [comment]
Where do you usually get the assets? Are they custom or is there a easier standard that it follows?
Example, trying to find bezel.bnp for joust. It has a transparent center for the screen.

Ive been editing in Gimp, but sure there is probably an easier way to gather instead of cutting/remaking.

**Reply by curif** [reply]
I look for assets in google images, for example, and I cut parts from photos or videos.
There are a lot of resources, but one that I consider great is https://www.arcade-museum.com/ for information, the database is extensive.
For images, bezels, art in general: https://www.arcadeartwork.org

I use Gimp too.

**Reply by WilsonBradley** [reply]
Awesome!!! thanks

**Comment by WilsonBradley** [comment]
Just found out via youtube, you already created Joust/ (where to get it?)

That's alright.. was practice.
